sonar-plugin-api/src/main/java/org/sonar/api/component/ResourcePerspectives.java

```diff
@@ -57,8 +57,10 @@
  * @see Testable
  * @see TestPlan
  * @since 3.5
+ * @deprecated since 5.2. Use appropriate methods on {@link SensorContext}.
  */
-public interface ResourcePerspectives extends Perspectives {
+@Deprecated
+public interface ResourcePerspectives {
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png)![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png)Do not forget to remove this deprecated code someday.[![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)